libs/cms/src/lib/cms.resolver.ts

```diff
@@ -347,13 +348,15 @@ export class CmsResolver {
 
   @Directive(cacheControlDirective())
   @Query(() => Article, { nullable: true })
-  getSingleArticle(
-    @Args('input') { lang, slug }: GetSingleArticleInput,
-  ): Promise<Article | null> {
-    return this.cmsElasticsearchService.getSingleDocumentTypeBySlug<Article>(
+  async getSingleArticle(@Args('input') { lang, slug }: GetSingleArticleInput) {
```

Comment:
Why remove the return type?

Reply:
The type is inferred but ok, I've added a type annotation now